go.sum

```diff
@@ -1,3 +1,5 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898/go.mod h1:Xbm+BRKSBEpa4q4hTSxohYNQpsxXPbPry4JJWOB3LB8=
```

Comment:
1335 new lines after go mod tidy?

Reply:
yup something off. Forgot to run tidy after deleting some packages for local testing or something.